keyboards/wsk/houndstooth/config.h

```diff
@@ -0,0 +1,59 @@
+#ifndef CONFIG_H
+#define CONFIG_H
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x6060
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    worldspawn
+#define PRODUCT         Houndstooth
+#define DESCRIPTION     Keyboard
+
+/* key matrix size */
+#define MATRIX_ROWS 8
+#define MATRIX_COLS 6
+
+/* key matrix pins */
+#define MATRIX_ROW_PINS { C6, F7, D7, B1, B4, B2, B5, B6 }
+#define MATRIX_COL_PINS { D1, F4, D0, F5, D4, F6 }
+#define UNUSED_PINS
+
+/* COL2ROW or ROW2COL */
+#define DIODE_DIRECTION COL2ROW
+
+/* number of backlight levels */
+
+//#ifdef BACKLIGHT_PIN
+//#define BACKLIGHT_LEVELS 0
+//#endif
+
+/* Set 0 if debouncing isn't needed */
+#define DEBOUNCING_DELAY 5
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+/* key combination for command */
+#define IS_COMMAND() ( \
+    keyboard_report->mods == (MOD_BIT(KC_LSHIFT) | MOD_BIT(KC_RSHIFT)) \
+)
+
+/* prevent stuck modifiers */
+#define PREVENT_STUCK_MODIFIERS
+
+
+#define RGB_DI_PIN F1
+//#ifdef RGB_DI_PIN
+//#define RGBLIGHT_ANIMATIONS
+#define RGBLED_NUM 0
```

Comment:
0? 😆 

May want to leave it undefined then. 

```suggestion
// #define RGBLED_NUM 0
```